library/include/ck/library/tensor_operation_instance/gpu/grouped_gemm_fixed_nk.hpp

```diff
@@ -97,6 +97,33 @@ void add_device_grouped_gemm_xdl_fixed_nk_f16_i8_f16_mk_nk_mn_instances(
                                                          PassThrough,
                                                          PassThrough>>>& instances);
 
+// bf16_inputA i8_inputB
+void add_device_grouped_gemm_xdl_fixed_nk_bf16_i8_bf16_mk_kn_mn_instances(
+    std::vector<std::unique_ptr<DeviceGroupedGemmFixedNK<Row,
```

Comment:
Please add conditional compilation if `CK_ENABLE_BF16` and `CK_ENABLE_I8` symbols are defined.